src/chat/Chat.js

```diff
@@ -17,26 +24,30 @@ export default class MainScreen extends React.Component {
       markAsRead, mute } = this.props;
 
     return (
-      <KeyboardAvoidingView style={styles.screen} behavior="padding">
+      <View style={styles.screen} >
         {!isOnline && <OfflineNotice />}
         {messages.length === 0 && caughtUp.older && caughtUp.newer &&
           <NoMessages narrow={narrow} />}
-        <MessageList
-          messages={messages}
-          narrow={narrow}
-          fetching={fetching}
-          caughtUp={caughtUp}
-          twentyFourHourTime={twentyFourHourTime}
-          subscriptions={subscriptions}
-          auth={auth}
-          fetchOlder={fetchOlder}
-          fetchNewer={fetchNewer}
-          doNarrow={doNarrow}
-          markAsRead={markAsRead}
-          mute={mute}
-        />
-        {canSendToNarrow(narrow) && <ComposeBox onSend={this.sendMessage} />}
-      </KeyboardAvoidingView>
+        <View>
+          <MessageList
+            messages={messages}
+            narrow={narrow}
+            fetching={fetching}
+            caughtUp={caughtUp}
+            twentyFourHourTime={twentyFourHourTime}
+            subscriptions={subscriptions}
+            auth={auth}
+            fetchOlder={fetchOlder}
+            fetchNewer={fetchNewer}
+            doNarrow={doNarrow}
+            markAsRead={markAsRead}
+            mute={mute}
+          />
+        </View>
+        <View style={moreStyles.composeBox}>
+          {canSendToNarrow(narrow) && <ComposeBox onSend={this.sendMessage} />}
```

Comment:
Are the extra Views necessary?
The one around MessageList may not be.
And the one around ComposeBox - you probably can apply the styles directly to the ComposeBox component?